client/karma.js

```diff
@@ -5,6 +5,7 @@ var util = require('../common/util')
 function Karma (socket, iframe, opener, navigator, location) {
   var startEmitted = false
   var reloadingContext = false
+  var socketReconnect = false
```

Comment:
Could you add a description for this variable?

Reply:
👍